services/121-service/src/sovrin/create-connection/dto/bulk-import.dto.ts

```diff
@@ -0,0 +1,21 @@
+import { IsNotEmpty, IsString, Length } from 'class-validator';
+import { ApiModelProperty } from '@nestjs/swagger';
+
+export class BulkImportDto {
+  @ApiModelProperty()
+  @IsNotEmpty()
+  @IsString()
+  @Length(10, 14)
```

Comment:
no, not really, I removed this check, because we have checks at other/better places already. Which for example also remove spaces, which would bump the allowed length to much more than 14 anyway..

I'm thinking now by the way if the phone-number check should return the incorrect phone numbers in the feedback, instead of just the count of that..